### PR TITLE
Add UK behavioural responses post

### LINKED ISSUE #2190: UK behavorial response post
I wrote a post about how we enhanced our modelling with behavioural responses, including income, substitution and capital gains elasticities.

### PR DESCRIPTION
**Pull request by vahid-ahmadi**
I wrote a post about how we enhanced our modelling with behavioural responses, including income, substitution and capital gains elasticities.

Fixes #2190

### CODE REVIEW

**Review by nikhilwoodruff — COMMENTED** [review]
LGTM- holding off for Max's review

File: src/posts/posts.json
Comment (MaxGhenis) [inline src/posts/posts.json:852]:
```suggestion
    "tags": ["uk", "policy", "behavioural-responses", "featured"],
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]
main suggestions/comments in the [doc](https://docs.google.com/document/d/1_073b_pz6p4HubGT06VX_TuEN_YpDXzDHOi2gTqVSbc/edit?tab=t.0)

File: src/images/posts/behavorial-post.jpg
Comment (MaxGhenis) [inline src/images/posts/behavorial-post.jpg:1]: rename `behavioural-responses.jpg`

File: src/posts/articles/behavioural-response-post.md
Comment (MaxGhenis) [inline src/posts/articles/behavioural-response-post.md:1]: rename `behavioural-responses.md`

File: src/posts/posts.json
Comment (MaxGhenis) [inline src/posts/posts.json:850]:
```suggestion
    "description": "PolicyEngine has enhanced its modelling with behavioural responses regarding earnings and capital gains.",
```

**Review by MaxGhenis — APPROVED** [review]
good to merge after these changes

**Pull request merged**
